src/Datagrid/Datagrid.php

```diff
@@ -189,6 +189,18 @@ public function hasActiveFilters(): bool
         return false;
     }
 
+    public function hasDisplayableFilters(): bool
```

Comment:
I'm not sure if we should add this method. Showing a filter or not is logic for the UI, not the general lib

Reply:
What is the purpose of the library ?

hasActiveFilters and hasDisplayableFilters are really similar. I think it's better to have the two methods in the same Interface/class.

Currently we have a loooot of duplicated code/really similar code. I want to deal with this, and I think we'll have to make some small trade-off.

There are some others UI method in this bundle, like Filter::getRenderSettings, or getTranslationDomain. The Datagrid class is also mainly a UI class, and is never use by others bundle. They only use the Pager and the ProxyQuery.